src/Console/HandlesConsoleInput.php

```diff
@@ -27,22 +29,16 @@ public function ask(
         ?array $options = null,
         ?string $default = null,
     ): string {
-        $questionString = ConsoleStyle::BG_YELLOW($question);
-
-        if ($options) {
-            $questionString .= ' [' . ConsoleStyle::FG_BLUE(
-                implode(
-                    ',',
-                    array_map(
-                        fn (string $option) => $option === $default ? strtoupper($option) : $option,
-                        $options,
-                    ),
-                ),
-            )
-                . '] ';
-        }
-
-        $this->output->write($questionString);
+        $message = ConsoleOutputBuilder::new(" ")
+            ->error(" $question ")
+            ->when(! ! $options, function (ConsoleOutputBuilder $builder) use ($options, $default) {
```

Comment:
I guess the double `!!` is to convert it to booleans? Why not `$options !== null` ?